src/core/util/memory_usage.h

```diff
@@ -260,7 +260,7 @@ size_t MemoryUsageOf(const T& x) {
 template <typename... Args>
 size_t MemoryUsageOf(const std::tuple<Args...>& t) {
   return std::apply(
-      [](const auto&... args) { return (MemoryUsageOf(args) + ...); }, t);
+      [](const auto&... args) { return (MemoryUsageOf(args) + ... + 0); }, t);
```

Comment:
For my understanding, why was this change needed?

Reply:
this handles tuple<> too